library/core/src/ptr/mut_ptr.rs

```diff
@@ -2415,13 +2528,15 @@ mod verify {
     generate_mut_arithmetic_harness!(i64, check_mut_add_i64, add);
     generate_mut_arithmetic_harness!(i128, check_mut_add_i128, add);
     generate_mut_arithmetic_harness!(isize, check_mut_add_isize, add);
+    generate_mut_arithmetic_harness!(u8, check_mut_add_u8, add);
```

Comment:
If we add this line, the comment below should be removed and the issue solved.

Reply:
Line removed in newest commit.